docs/integrations.md

```diff
@@ -0,0 +1,9 @@
+# Integrations 
```

Comment:
Can we add FontMatter as we have for other pages? This will expose this to website as well (: 

See https://thanos.io/contributing/how-to-contribute-to-docs.md/ for details (: